src/SmartComponents/CompletedTaskDetails/Columns.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { renderComponent } from '../../Utilities/helpers';
+
+const SystemNameCell = ({ /*id,*/ display_name }, index) => (
+  <Link
+    key={`system-title-${index}`}
+    //to={`/executed/${id}`}
+    to={`/`}
+  >
+    {display_name}
+  </Link>
+);
+
+SystemNameCell.propTypes = {
+  id: propTypes.string,
+  display_name: propTypes.string,
+};
+
+export const System = {
```

Comment:
I think renaming these column names to have 'Column' string appended at the end would make the code more readable (for example: System => SystemColumn).

Reply:
I'm not sure I understand what you mean here.

Reply:
We usually name React components in this manner with capital letters. When I saw the Systems object on line #21, I immediately thought that it should be component. Thus, I suggested to rename it to SystemColumn, to easly say that this is not React Component.

Or you can easly  make it lower case.

Reply:
ahh, I see. The `System => SystemColumn` threw me off. I thought you wanted me to set up the function to look like that and I didn't understand. :)